assets/src/design-system/theme/colors.js

```diff
@@ -157,6 +157,7 @@ const darkTheme = {
     positiveNormal: brand.green[40],
     positiveHover: brand.green[30],
     positivePress: brand.green[20],
+    activeShadow: opacity.white16,
```

Comment:
👍  this is a good add to the theme, I would say we should perhaps start a new section called "shadow" and nest it there (ie: `theme.colors.shadow.active`) since the interactiveBg eludes to the colors on interacting elements as opposed to flourishes. Also to keep this object from becoming unruly.